### DISCUSSION TITLE
METASPLOIT

### DISCUSSION THREAD

**Discussion by robel211** [rapid7/metasploit-framework#15307] [Q&A] [answered]
can i use metasploit for testing on other device connected to other wireless network for example androids????
i mean i am running kali and can i use it for exploiting other device on other network?
answers are much helpful!!!

AND how? for example i create the payload for android the (reverse_tcp) then should i send it to the victim and it is done?

**Comment by bcoles** [comment, marked as answer]
yes

**Reply by robel211** [reply]
thank you for responding sir!!
how? for example i create the payload for android the (reverse_tcp) then should i send it to the victim and it is done?

**Reply by bcoles** [reply]
Most Metasploit payloads use either TCP or UDP.

For reverse payloads, such as `reverse_tcp`, a network route must exist to route traffic from the compromised device to the Metasploit handler.

**Reply by robel211** [reply]
what do you mean by network route?
how do i create the route?

**Reply by robel211** [reply]
what do you mean by network route?

**Reply by robel211** [reply]
sorry i sent it twice!

**Reply by robel211** [reply]
.....

**Reply by robel211** [reply]
how!!!

**Reply by busterb** [reply]
I think you would save a lot of time for this team and yourself if you just did a little searching first! The concept of a network route might be new to you, but it's kind of like asking what concrete is on a bridge-building forum. This just isn't the place for it.

There are thousands of resources online that would do a much better job of explaining networking basics than a trouble ticket here could convey. Do a little reading and research, please!

**Reply by busterb** [reply]
After a quick search, this doesn't look like a bad thing to start with : https://www.youtube.com/watch?v=QKfk7YFILws

**Reply by robel211** [reply]
thanks sorry if i did mistake!!
